charts/azure-js/templates/deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.Release.Name}}-js
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{.Release.Name}}-js
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  minReadySeconds: 5 
+  template:
+    metadata:
+      labels:
+        app: {{.Release.Name}}-js
+    spec:
+      nodeSelector:
+        kubernetes.io/arch: amd64
+        kubernetes.io/os: linux
+      containers:
+      - name: {{.Release.Name}}-js
+        image: akashkeshari3010/arc-conformance-test:jsappv1
```

Comment:
Let's use the Arc ACR repo here. Pipeline YAML we can check in this PR or the next one, but till then we can consider manually building and pushing the image.

Reply:
Yes i will update the image in the pipeline PR